distribution/docs/src/main/resources/_contents/_sources/solrcloud-contents.adoc

```diff
@@ -0,0 +1,109 @@
+==== Solr Cloud
+
+Solr Cloud is a cluster of Solr Server instances that are fault tolerant and highly available.
+Each Solr Server instance in Solr Cloud provides a Solr Admin UI.
+
+Synchronization between Solr Server instances is managed by Zookeeper.
```

Comment:
Are you sure these two sentences are correct?  I thought Zookeeper was only used to store shared configuration and leader election but Solr is the one that actually makes decisions based on the Zookeeper state.